### PR TITLE
Support multiple parametrs on CONFIG SET

### LINKED ISSUE #1936: Add support for redis command: CONFIG SET - set multiple parameters in one call (R7)

### PR DESCRIPTION
**Pull request by Avital-Fine**
### Description

[CONFIG SET](https://redis.io/commands/config-set)
Closes #1936 

---

### Checklist

- [ ] Does `npm test` pass with this change (including linting)?
- [ ] Is the new or changed code fully tested?
- [ ] Is a documentation update included (if this change modifies existing APIs, or introduces new ones)?

### CODE REVIEW

File: packages/client/lib/commands/CONFIG_SET.ts [hunk 1]
```diff
@@ -1,5 +1,15 @@
-export function transformArguments(parameter: string, value: string): Array<string> {
-    return ['CONFIG', 'SET', parameter, value];
+export function transformArguments(
+    parameter: string,
+    value: string,
+    mulitipleParams?: Array<[parameter: string, value: string]>
+): Array<string> {
```
Comment (Avital-Fine) [inline packages/client/lib/commands/CONFIG_SET.ts:5]: @leibale Is there a better way to implement it without breaking the code? :sweat_smile:

File: packages/client/lib/commands/CONFIG_SET.ts [hunk 2]
```diff
@@ -1,5 +1,15 @@
...
+    const args = ['CONFIG', 'SET', parameter, value];
+
+    if (mulitipleParams) {
+        mulitipleParams.forEach(pair => args.push(pair[0], pair[1]));
```
Comment (mrbaguvix) [inline packages/client/lib/commands/CONFIG_SET.ts:9]: should we destructure for clarity?
```suggestion
        mulitipleParams.forEach(([parameter, value]) => args.push(parameter, value));
```

Comment (leibale) [reply packages/client/lib/commands/CONFIG_SET.ts]: @Avital-Fine checkout my commit... :)

File: packages/client/lib/commands/CONFIG_SET.ts (hunk 1 above)
Comment (leibale) [reply packages/client/lib/commands/CONFIG_SET.ts]: @Avital-Fine checkout my commit... :)

**Review by leibale — APPROVED** [review]

**Pull request merged**